steps/mentor-calibre-drc/configure.yml

```diff
@@ -36,6 +36,7 @@ commands:
 
 parameters:
   design_name: undefined
+  ruleset_name: calibre-drc-block.rule
```

Comment:
Can you add a comment that shows how it is used?

```
parameters:
  design_name: undefined
  # Use the rule deck "inputs/adk/${drc_rule_deck}"
  drc_rule_deck: calibre-drc-block.rule
```

Maybe "drc_rule_deck" instead of "ruleset_name"?

Reply:
done